android_app/app/src/main/java/com/health/openscale/gui/measurement/ChartMeasurementView.java

```diff
@@ -395,7 +399,18 @@ public void refreshMeasurementList() {
         }
 
         if (prefs.getBoolean("trendLine", false)) {
-            addTrendLine(lineDataSets);
+            switch (prefs.getString("trendlineComputationMethod", "Exponentially Smoothed Moving Average")) {
```

Comment:
would be better to work with ENUM variable.

Reply:
I am not sure how to connect enums and Android preferences, can you give any advice or example for that? Currently, I added a separate class with two static Strings which contain the values I specified in the `arrays.xml` file.

Reply:
I am not sure either. Anyway I just delete the separate class and put the two static string to ChartMeasurementView.